pages/contact.html

```diff
@@ -26,20 +27,15 @@
                 <div class="ui padded basic segment">
                     <h1 class="ui block header center blue aligned">Contact Details</h1>
                     <div class="ui center aligned basic segment">
-                        <p>Email School:<a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
-                        <p>Web: mountsioncbssecondary.ie</p>
-                        <p>Telephone Number: 051 377378</p>
-                        <p>Fax Number: 051-376468</p>
-                        <p>School Roll No. 64930i</p>
-                        <p>Address:</p>
-                        <p>Mount Sion CBS Secondary School,</p>
+                        <p><a href="mailto:info@mountsioncbssecondary.ie"><i class="envelope link icon"></a></i>info@mountsioncbssecondary.ie</p> 
+                        <p><a target="_blank" href="http://www.mountsioncbssecondary.ie"><i class="globe icon"></i></a>mountsioncbssecondary.ie</p>
+                        <p><a href="tel:+35351377378"><i class="phone icon"></i></a>051 377378</p>
+                        <p><a href="tel:+35351376468"><i class="fax icon"></a></i>051 376468</p>
```

Comment:
For the fax we probably don't need to provide a telephone link as people can't ring the fax number. The icon looks great though.

Reply:
OK, so you are saying just to have the icon, not the hyperlink, right?

Reply:
Yes exactly.

Reply:
@evanhearne after the review the fax number is still a link

Reply:
@laurafitzgerald amended this.